### PR TITLE
docs(switch): Revamp Overview

### PR DESCRIPTION
**Pull request by ntacheva**

### CODE REVIEW

File: components/switch/overview.md [hunk 1]
```diff
@@ -12,10 +12,10 @@ position: 0
 
 The <a href = "https://www.telerik.com/blazor-ui/switch" target="_blank">Blazor Switch component</a> allows the user to toggle between checked and unchecked states.
 
-#### To use a Telerik Switch for Blazor
+## Creating Switch for Blazor
```
Comment (dimodi) [inline components/switch/overview.md:15]:
```suggestion
## Creating Blazor Switch
```

File: components/switch/overview.md [hunk 2]
```diff
@@ -12,10 +12,10 @@ position: 0
...
 
-1. add the `TelerikSwitch` tag
-1. provide `Value` (one-way data binding) or `bind-Value` (two-way data binding) property
+1. Use the `TelerikSwitch` tag
+1. Provide `Value` (one-way data binding) or `bind-Value` (two-way data binding) property
```
Comment (dimodi) [inline components/switch/overview.md:18]: Or you can just say "Set the `Value` parameter to a `bool` property. It supports one-way or two-way binding."
```suggestion
1. Provide `Value` (one-way data binding) or `@bind-Value` (two-way binding) property
```

File: components/switch/overview.md [hunk 3]
```diff
@@ -37,116 +37,78 @@ The <a href = "https://www.telerik.com/blazor-ui/switch" target="_blank">Blazor
...
+## Validation
 
-* `Width` - the width of the component.
+You can ensure that the component value is acceptable by using the built-in validation. [Read more about input validation....]({%slug common-features/input-validation%}) 
 
-* `TabIndex` - the `tabindex` attribute rendered on the Switch.
+## Switch Parameters
 
-* `Value` and `bind-Value` -  accept `bool` and `bool?` data types.
+The Switch provides the following features to allow further customization of its behavior:
```
Comment (dimodi) [inline components/switch/overview.md:54]: Also link the API page

File: components/switch/overview.md [hunk 4]
```diff
@@ -37,116 +37,78 @@ The <a href = "https://www.telerik.com/blazor-ui/switch" target="_blank">Blazor
...
-* `OnLabel` - the label of the component when the `Value` is `true`.
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
 
-* `OffLabel` - the label of the component when the `Value` is `false`.
+| Attribute | Type and Default Value | Description |
+|----------|----------|----------|
+| `Enabled` | `bool` | whether the component is enabled.
+| `Id` | `string` | renders as the `id` attribute on the wrapping `<span>` element of the component.
+| `TabIndex` | `int` | the `tabindex` attribute rendered on the Switch.
+| `Value` and `bind-Value` | `bool` or `bool?`| the value of the Switch.
```
Comment (dimodi) [inline components/switch/overview.md:63]: @bind-Value is not a separate parameter that requires mentioning. "@bind" is a Blazor directive. We can specify that it supports two-way binding in the description.

File: components/switch/overview.md [hunk 5]
```diff
@@ -37,116 +37,78 @@ The <a href = "https://www.telerik.com/blazor-ui/switch" target="_blank">Blazor
...
-}
-
-<TelerikSwitch @bind-Value="@hasAgreed"
-               Id="mySwitch"
-               OnLabel="@myOnLabel"
-               OffLabel="@myOffLabel"
-               Width="100px">
-</TelerikSwitch>
-I agree to the terms and conditions
+<TelerikSwitch @bind-Value="@toggleSwitch" @ref="@TheSwitchRef" />
```
Comment (dimodi) [inline components/switch/overview.md:95]: Should be `SwitchRef` - see the Docs & Demos Style Guide in the blazor repo wiki

**Review by dimodi — CHANGES_REQUESTED** [review]
Some comments related to consistency and docs coding style guide.

File: components/switch/overview.md [hunk 6]
```diff
@@ -12,10 +12,10 @@ position: 0
 
 The <a href = "https://www.telerik.com/blazor-ui/switch" target="_blank">Blazor Switch component</a> allows the user to toggle between checked and unchecked states.
 
-#### To use a Telerik Switch for Blazor
+## Creating Switch for Blazor
 
-1. add the `TelerikSwitch` tag
-1. provide `Value` (one-way data binding) or `bind-Value` (two-way data binding) property
+1. Use the `TelerikSwitch` tag
```
[the remark this answers is not in the document]
Comment (ntacheva) [reply components/switch/overview.md:17]: I think `<Foo>` tag is better. Updated

**Review by dimodi — APPROVED** [review]

**Pull request merged**